.github/workflows/test.yml

```diff
@@ -154,37 +156,94 @@ jobs:
         terraform init
         terraform plan -out=terraform.plan
         terraform apply -auto-approve terraform.plan
+    - name: Install hcledit
+      run: |
+        hcledit_version="0.2.9"
+        hcledit_artifact=hcledit_${hcledit_version}_linux_amd64.tar.gz
+        curl -fsSL -o "${hcledit_artifact}" "https://github.com/minamijoyo/hcledit/releases/download/v${hcledit_version}/${hcledit_artifact}"
+        tar xvzf "${hcledit_artifact}"
+        sudo mv hcledit /usr/local/bin/ && rm "${hcledit_artifact}" && hcledit version
     - name: Setup terraform-aws-eks conversion process
       if: (contains(github.event.pull_request.labels.*.name, 'deploy-test') || github.ref == 'refs/heads/master')
       working-directory: ./convert
       run: |
         pip install -r requirements.txt
         ssh-keygen -t rsa -f dummy.pem -N ''
-        ./convert.py print-stack --stack-name $NAME --region $AWS_REGION --verbose --yaml > stack-data.yaml
-        ./convert.py create-tfvars --stack-name $NAME --region $AWS_REGION --ssh-key-path ./dummy.pem > terraform/terraform.tfvars.json
-        ./convert.py get-imports --stack-name $NAME --region $AWS_REGION > terraform/imports.sh
+
+        ##TODO
```

Comment:
Assume this is vestigial at this point?

Reply:
Ultimately we will point to the released module